MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,489 @@
+
+Also, The custom editor value can be validated manually by using [OnValidateValue] method on custom editor `Value` or `Focus changed` event which is used to validate the custom editor value based on data form [ValidationMode] set.
+
+## Commit mode
+
+The [CommitMode] determines when the value should be committed to the data object.
+
+The supported commit modes are as follows:
+
+* LostFocus
+* PropertyChanged
+* Manual
+
+### LostFocus
+If the commit mode is LostFocus, the value is committed when the editor lost its focus.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage  xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        CommitMode="LostFocus">
+    </DataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+this.DataForm.CommitMode = DataFormCommitMode.LostFocus;
+{% endhighlight %}
+{% endtabs %}
+
+
+### PropertyChanged
+If the commit mode is PropertyChanged, the value will be committed immediately when it is changed.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage  xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        CommitMode="PropertyChanged">
+    </DataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+this.DataForm.CommitMode = DataFormCommitMode.PropertyChanged;
+{% endhighlight %}
+{% endtabs %}
+
+### Manual
+If the commit mode is Manual, the value should be committed manually by calling the [SfDataForm.Commit] method.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage  xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        CommitMode="Manual">
+    </DataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+this.DataForm.CommitMode = DataFormCommitMode.Manual;
+{% endhighlight %}
+{% endtabs %}
+
+The following code commits the value of all the properties in the data object:
+
+{% tabs %}
+{% highlight c# %}
+this.DataForm.Commit();
+{% endhighlight %}
+{% endtabs %}
```

Comment:
mention note: On manula commit call manual validation will be called to validate the properties before commit